source/guide/3.2/ui-login.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bt_wiki
+title: Login
+category: User Interface
+publish: true
+abstract: Login Reference
+pageord: 90
+
+---
+
+{%summary%} {{page.abstract}}{%endsummary%}
+{%summary%}This is an overview of the login screen User Interface{%endsummary%}.
+
+# Overview
+Version 3.2 introduced security support and so, it is possible to bootstrap the manager with security configuration or without it.
+
+# Login Screen
+After bootstrapping the manager with security support, the UI will redirect the users to the Login screen.<br>
+The standard `Username` and `Password` are required as well as the option to check `Remember me`. <br>
+
+![The Login Screen](/guide/images/ui/ui-login-screen.png)
+
+# Logout
+Once logged in, the user name is displayed on the top right of the screen, right next to the `Log Out` button. <br>
+The user can opt to logout anytime by simply clicking on the `Log Out` (as shown below). <br>
```

Comment:
Not sure I understand what you want me to do here...
It's already written "on the Log Out button"